### PR TITLE
[BACKPORT 0.2] `hc` cli documentation

### PR DESCRIPTION
**Pull request by pdaoust**
### Summary

Backporting #2100 to 0.2.

### TODO:
- [x] CHANGELOG(s) updated with appropriate info
- [ ] Just before pressing the merge button, ensure new entries to CHANGELOG(s) are still under the _UNRELEASED_ heading

### CODE REVIEW

**Review by ThetaSinner — APPROVED** [review]

File: crates/hc/README.md [hunk 1]
```diff
@@ -2,44 +2,30 @@
...
+This command gives you a suite of tools for developing, inspecting, executing, and testing your Holochain apps. Some top-level subcommands are implemented as separate crates, and others are separate binaries -- commands whose names start with `hc-` and are automatically made available as subcommands if they exist in your shell's path. Here is a list of all available subcommands:
 
-FLAGS:
-    -h, --help       Prints help information
-    -V, --version    Prints version information
+- `hc dna`, `hc app`, and `hc web-app` scaffold, bundle, and unbundle DNAs, hApps and web hApps respectively. See [holochain_cli_bundle](https://github.com/holochain/holochain/tree/develop/crates/hc_bundle) for more info.
+- `hc sandbox` creates and executes temporary or persistent conductor configurations for you to run test instances of your hApp with. See [holochain_cli_sandbox](https://github.com/holochain/holochain/tree/develop/crates/hc_sandbox) for more info.
+- `hc signal-srv` runs a local WebRTC signal server for peers to establish connections with each other. See [holochain_cli_signal_srv](https://github.com/holochain/holochain/tree/develop/crates/hc_signal_srv) for more info.
```
Comment (neonphog) [inline crates/hc/README.md:11]: This doesn't exist anymore. It's been renamed to `run-local-services`.

File: crates/hc_signal_srv/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,69 @@
+use clap::Parser;
```
Comment (neonphog) [inline crates/hc_signal_srv/src/lib.rs:1]: I think you have a merge issue here, this file shouldn't exist.

I also notice this is true for develop, maybe your non-backport PR had this same issue?

File: crates/hc_run_local_services/src/lib.rs [hunk 3]
```diff
@@ -1,44 +1,44 @@
...
     /// If set, write the bound address list to a new file, separated by
     /// newlines. If the file exists, an error will be returned.
-    #[structopt(long)]
+    #[arg(long)]
     signal_address_path: Option<std::path::PathBuf>,
 
-    /// A comma separated list of interfaces on which to run the signal server.
-    #[structopt(long, default_value = "127.0.0.1")]
+    /// A comma-separated list of interfaces on which to run the signal server.
+    #[arg(long, default_value = "127.0.0.1", value_delimiter = ',')]
```
Comment (neonphog) [inline crates/hc_run_local_services/src/lib.rs:32]: I don't think `value_delimiter` has any effect if the type is `String`... or does it add some hint to the help text?

**Review by neonphog — CHANGES_REQUESTED** [review]
The reference to signal-srv in the hc README, and the merge issue bringing back the `hc_signal_srv/src/lib.rs` should be fixed (and possibly fixed in `develop` also.

Otherwise, this looks awesome! Nice work :guitar:

File: crates/hc/README.md (hunk 1 above)
Comment (pdaoust) [reply crates/hc/README.md:11]: dang, missed that in the text. Thanks.

File: crates/hc_run_local_services/src/lib.rs (hunk 3 above)
Comment (pdaoust) [reply crates/hc_run_local_services/src/lib.rs:32]: I just noticed that it was meant to be comma-separated and remembered another instance where the arg wasn't working properly for lack of this property. Now I see it's a string, not a `Vec<String>`, and assuming it's parsed into a vec elsewhere?

Comment (pdaoust) [reply crates/hc_run_local_services/src/lib.rs]: removed the property

File: crates/hc/README.md [hunk 4]
```diff
@@ -2,44 +2,30 @@
...
 
-FLAGS:
-    -h, --help       Prints help information
-    -V, --version    Prints version information
+- `hc dna`, `hc app`, and `hc web-app` scaffold, bundle, and unbundle DNAs, hApps and web hApps respectively. See [holochain_cli_bundle](https://github.com/holochain/holochain/tree/develop/crates/hc_bundle) for more info.
+- `hc sandbox` creates and executes temporary or persistent conductor configurations for you to run test instances of your hApp with. See [holochain_cli_sandbox](https://github.com/holochain/holochain/tree/develop/crates/hc_sandbox) for more info.
+- `hc run-local-services` runs local bootstrap and WebRTC signal servers for peers to establish connections with each other. This is useful for running tests. See [holochain_cli_run_local_services](https://github.com/holochain/holochain/tree/develop/crates/hc_run_local_services) for more info.
```
Comment (pdaoust) [inline crates/hc/README.md:11]: @neonphog here's the updated readme text.

**Comment by pdaoust** [comment]
Thanks for the review @neonphog . I've got the requested fixes in, and I've also got them in a new PR #2363 against `develop`. Requesting a follow up review to un-block. Thanks!

**Review by neonphog — APPROVED** [review]
:+1:

**Pull request merged**